crates/agglayer-tries/Cargo.toml

```diff
@@ -12,6 +12,7 @@ workspace = true
 [dependencies]
 agglayer-primitives.workspace = true
 
+
```

Comment:
Super-nitty nit: why the extra blank space?

Reply:
leftover of a dependency round trip 😅